source/documentation/other-topics/rotate-tokens-keys.html.md.erb

```diff
@@ -0,0 +1,31 @@
+---
+title: Rotating tokens and keys
+last_reviewed_on: 2021-04-19
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+
+## Overview
+
+Below is the guidance for rotating the tokens and keys that are set as environment variables for your application container or your automated deployment pipeline. 
+This include only the variables fetched from system secrets from a namespace and not include any application related variables.
```

Comment:
```suggestion
This guide covers only secrets provided by the Cloud Platform system (your application may have other secrets too).
```

Reply:
Thanks @davidread . I have updated the lines